src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## Unreleased
 
+## 1.8.0-rc.1
+
+Released 2024-Mar-27
+
+* `TracerProvider`s can now have a sampler configured via the
```

Comment:
Why this needs to be moved from beta.1 to rc.1?

Reply:
See #5480